src/runtime/hexagon_remote/sim_host.cpp

```diff
@@ -84,6 +84,15 @@ int init_sim() {
                 printf("HexagonWrapper::SetTracing failed: %d\n", status);
                 return -1;
             }
+        } else {
+            const char *T = getenv("HL_HEXAGON_SIM_MEM_TRACE");
+            if (T && T[0] != 0) {
```

Comment:
Whups, I missed that part. Probably because I didn't notice that T was passed as an argument, because it wasn't lowercase. :-)